Sources/iPad/View Controllers/Social/ActivityDetailViewController_iPad.m

```diff
@@ -63,6 +63,17 @@ - (void)viewDidLoad
     self.tblvActivityDetail.backgroundView = [[[CustomBackgroundView alloc] initWithFrame:CGRectZero] autorelease];
     _navigation.topItem.title = Localize(@"Details");
     _navigation.titleTextAttributes = [NSDictionary dictionaryWithObject:[UIColor whiteColor] forKey:UITextAttributeTextColor];
+    UIButton * topViewButton = [[UIButton alloc] initWithFrame:CGRectMake(100,0,_navigation.frame.size.width-200,20)];
+    [topViewButton addTarget:self action:@selector(scrollTableViewToTop:) forControlEvents:UIControlEventTouchUpInside];
+    
+    topViewButton.backgroundColor = [UIColor clearColor];
+    [_navigation addSubview:topViewButton];
+    
+    
+}
+
+-(void) scrollTableViewToTop:(id) sender {
+    [self.tblvActivityDetail setContentOffset:CGPointZero animated:YES];
```

Comment:
The new code above is duplicated in few files. Find a way to avoid this duplication. E.g. define a custom `ExoTapToScrollButton` with all the necessary code to handle the tap, with a protocol that defines the method `scrollTableViewToTop`.